### PR TITLE
Release 185.0.0

### PR DESCRIPTION
**Pull request by bergeron**
Minor release of `network-controller`

### CODE REVIEW

File: packages/network-controller/CHANGELOG.md
Comment (Gudahtt) [inline packages/network-controller/CHANGELOG.md:1]: This is missing a few changes:

* https://github.com/MetaMask/core/pull/4566
* https://github.com/MetaMask/core/pull/4576
* https://github.com/MetaMask/core/pull/4584
* https://github.com/MetaMask/core/pull/4607 (minimum version of dependency was bumped)

File: packages/network-controller/CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - `removeNetworkConfiguration` now throws an error if you attempt to remove the selected network ([#4566](https://github.com/MetaMask/core/pull/4566))
```
Comment (Gudahtt) [inline packages/network-controller/CHANGELOG.md:14]: This is probably better documented under "Fixed", as suggested in the PR description for this change

File: packages/network-controller/CHANGELOG.md
Comment (bergeron) [reply packages/network-controller/CHANGELOG.md:1]: Thanks.  4566 should already have been there, but I mentioned the other 3

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by sleepytanya — APPROVED** [review]

**Pull request merged**